examples/dataset/object_detection_3d/object_detection_3d/upload_dataset.py

```diff
@@ -0,0 +1,129 @@
+# are used to avoid penalizing inferences labeled as "Car" and "Pedestrian" respectively.
+SUPPORTED_LABELS = ["Car", "Pedestrian", "Cyclist", "Van", "Person_sitting", "DontCare"]
+LABEL_FILE_COLUMNS = {
+    "image_id",
+    "left_image",
+    "right_image",
+    "velodyne",
+    "objects",
+    "P0",
+    "P1",
+    "P2",
+    "P3",
+    "R0_rect",
+    "Tr_velo_to_cam",
+    "Tr_imu_to_velo",
+}
+
+
+def load_data(df_raw: pd.DataFrame) -> pd.DataFrame:
+    records = []
+    meta_cols = [col for col in df_raw.columns if col not in LABEL_FILE_COLUMNS]
+
+    for record in df_raw.itertuples():
+        bboxes_2d = [
+            LabeledBoundingBox(
+                label=box["label"],
+                top_left=tuple(box["left_top_left"]),
+                bottom_right=tuple(box["left_bottom_right"]),
+                occluded=box["occluded"],  # type: ignore[call-arg]
+                truncated=box["truncated"],  # type: ignore[call-arg]
+                alpha=box["alpha"],  # type: ignore[call-arg]
+                difficulty=box["difficulty"],  # type: ignore[call-arg]
+            )
+            for box in record.objects
+        ]
+        bboxes_3d = [
+            LabeledBoundingBox3D(
+                label=box["label"],
+                dimensions=(box["dim_x"], box["dim_y"], box["dim_z"]),
+                center=(box["loc_x"], box["loc_y"], box["loc_z"] + (box["dim_z"] / 2.0)),
+                # translate in z (up) axis to be in center
+                rotations=(0.0, 0.0, box["rotation_y"]),  # Z yaw axis for lidar coordinates
+                occluded=box["occluded"],  # type: ignore[call-arg]
+                truncated=box["truncated"],  # type: ignore[call-arg]
+                alpha=box["alpha"],  # type: ignore[call-arg]
+                difficulty=box["difficulty"],  # type: ignore[call-arg]
+            )
+            for box in record.objects
+        ]
+        counts = Counter([r["label"] for r in record.objects])
+        metadata = {col: getattr(record, col) for col in meta_cols}
+        records.append(
+            {
+                "image_id": record.image_id,
+                "locator": record.left_image,
+                "image_bboxes": bboxes_2d,
+                "right": ImageAsset(locator=record.right_image),
+                "velodyne": PointCloudAsset(locator=record.velodyne),
+                "total_objects": len(bboxes_3d),
+                "n_car": counts["Car"],
+                "n_pedestrian": counts["Pedestrian"],
+                "n_cyclist": counts["Cyclist"],
+                "velodyne_bboxes": bboxes_3d,
+                "velodyne_to_camera_transformation": record.Tr_velo_to_cam,
+                "camera_rectification": record.R0_rect,
+                "image_projection": record.P2,
+                **metadata,
+            },
+        )
+
+    return pd.DataFrame(records)
+
+
+def run(args: Namespace) -> int:
+    kolena.initialize(verbose=True)
```

Comment:
If we pin our `kolena` version to the latest release, we can skip the `initialize` call

(based on https://github.com/kolenaIO/kolena/pull/537)